### PR TITLE
Added example into initial-slides.md and filtered images folder from …

### PR DESCRIPTION
**Pull request by nmaas87**
…stash view to prevent errors / creation of images.md file to improve https://github.com/msoedov/hacker-slides/issues/23 :)

### CODE REVIEW

**Comment by servusoft** [comment]
The travis has a error:
https://travis-ci.org/msoedov/hacker-slides/builds/344923733?utm_source=github_status&utm_medium=notification

try to use `mkdir -p slides`  in the `.travis.yml` to prevent error:
https://github.com/msoedov/hacker-slides/blob/master/.travis.yml#L4

File: initial-slides.md [hunk 1]
```diff
@@ -69,6 +69,26 @@ services:
...
+Copy images into slides/images/ & include with MD:
+
+```
+![demoPicture](../../images/demo.png)
+
+```
+or HTML:
+
+```
+<img src="../../images/demo.png">
```
Comment (msoedov) [inline initial-slides.md]: I think 

```
<img src="/images/demo.png">
```
should work

File: main.go [hunk 2]
```diff
@@ -127,7 +127,9 @@ func NewApp() *gin.Engine {
 		}
 		var stash []string
 		for _, file := range files {
-			stash = append(stash, file.Name())
+      if file.Name() != "images" {
+			 stash = append(stash, file.Name())      
+      }
```
Comment (msoedov) [inline main.go]: 1 . Please run `go fmt`
2. I would make it
```go
if file.IsDir() {
      continue
}
stash = append(stash, file.Name())
```

**Comment by msoedov** [comment]
I would probably use another [dog/cat ](https://static.boredpanda.com/blog/wp-content/uploads/2017/04/cute-dog-shiba-inu-ryuji-japan-68.jpg)  pic in the example :)

**Comment by nmaas87** [comment]
I did change the requested things and you're right, /images works :)!

**Comment by servusoft** [comment]
I think, it will not works with `docker run -it -p 8080:8080 -v $(pwd)/slides:/app/slides msoedov/hacker-slides`. the `./slides` will be empty. the image should manually copied into `./slides/images`

**Comment by nmaas87** [comment]
@servusoft I actually did always mount my current directory with the slides into /app/slides. And now, I got my current directory and the subfolder images within that, which contains the images. Mounting the main folder / current directory will make it available as /app/slides as well as all subdirectorys of the current directory. So, images will be right were it is needed then :)

**Comment by msoedov** [comment]
The issue @servusoft referring to is when docker container runs outside git repo path and therefore the is no `slides/images/demo.png`  png image. I don't think this a critical issue though.  This example image could be placed into `/static` as well and it will fix this problem. LGTM thank you for the patch @nmaas87  man!

**Comment by nmaas87** [comment]
Thank you very much @msoedov and @servusoft for your inputs :). I am really glad to use this tool today during the https://piandmore.de/en conference. In fact, I have been working the whole week with it in different patched states and I am really glad about you, @msoedov making such an awesome job getting reveal to be used so easily (especially the PDF support as well, as that tends to be a pain with phantomjs ;)). So - thanks :)!

**Comment by servusoft** [comment]
I agree @nmaas87 too, :have fun on PI ++

**Pull request merged**